docs/pages/access-controls/reference.mdx

```diff
@@ -89,14 +89,20 @@ spec:
     max_sessions: 10
     # permit_x11_forwarding allows users to use X11 forwarding with openssh clients and servers through the proxy
     permit_x11_forwarding: true
-    # Specify whether or not to record the user's desktop sessions.
-    # Desktop session recording is enabled if one or more of the user's
-    # roles has enabled recording. Defaults to true if unspecified.
-    # Desktop sessions will never be recorded if auth_service.session_recording
-    # is set to 'off' in teleport.yaml or if the cluster's session_recording_config
-    # resource has set 'mode: off'.
+    # Session recording options.
     record_sessions:
+      # Specify whether or not to record the user's desktop sessions.
+      # Desktop session recording is enabled if one or more of the user's
+      # roles has enabled recording. Defaults to true if unspecified.
+      # Desktop sessions will never be recorded if auth_service.session_recording
+      # is set to 'off' in teleport.yaml or if the cluster's session_recording_config
+      # resource has set 'mode: off'.
       desktop: true
+      # Optional: Session recording mode default value used when sessions kind
```

Comment:
I'm not sure I understand this comment. Does it mean that if the `desktop` or `ssh` field is left out of the config, it will be set as `best_effort` or `strict`, depending on the value chosen here? If so, we should document that `best_effort` and `strict` are valid options for `desktop`. 

In addition, what do `best_effort` and `strict` indicate here? We should explain these terms at the top of `record_sessions` if they apply to each of the sub-fields within that field.

Reply:
> I'm not sure I understand this comment. Does it mean that if the desktop or ssh field is left out of the config, it will be set as best_effort or strict, depending on the value chosen here? If so, we should document that best_effort and strict are valid options for desktop.

Yes. But currently, `desktop` doesn't support `best_effort` or `strict`, so it would affect only `ssh` (I'll update to clarify).

> In addition, what do best_effort and strict indicate here? We should explain these terms at the top of record_sessions if they apply to each of the sub-fields within that field.

Do you think we should add it here also? I've created a section on another page to explain this. Not sure what is the best option here: copying the definition or referencing that page.

Reply:
If possible, I think we should copy the definition so a user doesn't need to navigate between pages and possibly shift context.

Reply:
Added the description on `record_sessions`.